plugins/heroku/src/setConfigVars.ts

```diff
@@ -1,5 +1,6 @@
 import type { Logger } from 'winston'
 import heroku from './herokuClient'
+import type { HerokuApiResGetRegion } from 'heroku-client'
 import { ToolKitError } from '@dotcom-tool-kit/error'
 import { VaultEnvVars, Environment } from '@dotcom-tool-kit/vault'
 import { HerokuApiResPipeline } from 'heroku-client'
```

Comment:
**nitpick**: this import can be merged with the other import from `heroku-client`

```suggestion
import { ToolKitError } from '@dotcom-tool-kit/error'
import { VaultEnvVars, Environment } from '@dotcom-tool-kit/vault'
import type { HerokuApiResGetRegion, HerokuApiResPipeline } from 'heroku-client'
```

Reply:
Thanks Ivo I've done this on the branch and merged it into my one `feat` commit